src/components/bottom-sheet/AnimatedBottomSheet.tsx

```diff
@@ -45,7 +45,7 @@ const useStyles = StyleSheet.createThemeHook((theme) => ({
     width: '100%',
     position: 'absolute',
     bottom: 0,
-    borderTopLeftRadius: theme.border.radius.regular,
-    borderTopRightRadius: theme.border.radius.regular,
+    borderTopLeftRadius: theme.border.radius.circle,
+    borderTopRightRadius: theme.border.radius.circle,
```

Comment:
Looking through the issue comments, didn't you settle on keeping the previous border radius here?

Reply:
Yes, however, I showed it to @ckbilstad and he said we can use that borders, even for other bottom sheets.